frontend/src/components/UI/Form/CreateBudgetLinesForm/CreateBudgetLinesForm.jsx

```diff
@@ -60,13 +63,13 @@ export const CreateBudgetLinesForm = ({
         warning: "warning"
     });
     const isFormComplete =
-        selectedCan && enteredDescription && enteredAmount && enteredMonth && enteredDay && enteredYear;
+        selectedCan && servicesComponentId && enteredAmount && enteredMonth && enteredDay && enteredYear;
 
     // validate all budgetline fields if in review mode and is editing
     if (isReviewMode && isEditing) {
         suite({
             selectedCan,
-            enteredDescription,
+            // enteredDescription,
```

Comment:
please remove commented out code

Reply:
better to add TODOs for adding validation later if needed